client/router_create.go

```diff
@@ -466,6 +466,11 @@ func ClusterRoles() []*rbacv1.ClusterRole {
 				Resources: []string{"nodes"},
 				Verbs:     []string{"get", "list", "watch"},
 			},
+			{
+				Verbs:     []string{"watch", "create", "patch"},
```

Comment:
Do we need this cluster scoped permission?

Reply:
Updated in 1b5d487c84b9ec4ead384eedf377225bd13e1936